src/middlewares/errorHandler.ts

```diff
@@ -0,0 +1,37 @@
+import { NextFunction, Request, Response } from "express";
+
+interface ResponseErrorInterface extends Error {
+    statusCode?: number;
+}
+
+interface HTTPResponseErrorInterface {
+    name: string;
+    statusCode: number;
+    message?: string;
+}
```

Comment:
I don't see the point of creating interfaces when they are not exported

Reply:
They are both used in the error handling middleware.